lib/sidekiq/client.rb

```diff
@@ -218,6 +219,7 @@ def normalize_item(item)
       item['queue'.freeze] = item['queue'.freeze].to_s
       item['jid'.freeze] ||= SecureRandom.hex(12)
       item['enqueued_at'.freeze] ||= Time.now.to_f
```

Comment:
That makes more sense! Pushed https://github.com/mrsimo/sidekiq/commit/b091d5c0542940780d9ddda765e0b8a1bd08ec02